app/javascript/components/widgets/widgets/forest-change/fires-alerts/component.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PureComponent } from 'react';
```

Comment:
As this is identical, could we import this into from the glads widget?

Reply:
Or even better make a new shared component in the `/widgets/components` folder?

Reply:
Yeah we definitely could.